### ISSUE TITLE
Severly broken rendering of 3d images

### ISSUE THREAD

**Issue by clecap** [mozilla/pdf.js#12570]
Attach (recommended) or Link to PDF file here:
[pdf-bugs.pdf](https://github.com/mozilla/pdf.js/files/5483662/pdf-bugs.pdf)

Configuration:
- Web browser and its version: Chrome 86.0.4240.80
- Operating system and its version: MacOS 10.15.6
- Is a browser extension: Chrome browser extension  2.3.164

The problems show up in the PDF Viewer extension in Chrome. 

When I remove the PDF Viewer extension and have it displayed by the native features of the browser, everything is fine. Also Firefox (without PDF Viewer extension) has no problems.

The rendering errors depend on the specific magnification size and might look different and even worse than with my examples, when changing the magnification.

What is the expected behavior? (add screenshot)
![Screenshot 2020-11-03 at 20 31 57](https://user-images.githubusercontent.com/7084049/98032383-39a18d00-1e14-11eb-9362-bacfa77b63ab.png)

![Screenshot 2020-11-03 at 20 31 30](https://user-images.githubusercontent.com/7084049/98032493-63f34a80-1e14-11eb-823b-d81c07c6b65a.png)

![Screenshot 2020-11-03 at 20 30 45](https://user-images.githubusercontent.com/7084049/98032548-753c5700-1e14-11eb-832f-252af6e166ac.png)

![Screenshot 2020-11-03 at 20 30 02](https://user-images.githubusercontent.com/7084049/98032695-a7e64f80-1e14-11eb-829a-46a11d501847.png)

What went wrong? (add screenshot)
<img width="490" alt="Screenshot 2020-11-03 at 20 31 48" src="https://user-images.githubusercontent.com/7084049/98032404-41f9c800-1e14-11eb-9664-178efa8a72a4.png">

<img width="338" alt="Screenshot 2020-11-03 at 20 31 17" src="https://user-images.githubusercontent.com/7084049/98032521-6c4b8580-1e14-11eb-8ac3-904b68cdfe3c.png">

<img width="558" alt="Screenshot 2020-11-03 at 20 30 38" src="https://user-images.githubusercontent.com/7084049/98032571-7cfbfb80-1e14-11eb-8c07-9bbab3f2dbd9.png">
![Screenshot 2020-11-03 at 20 30 23](https://user-images.githubusercontent.com/7084049/98032585-82f1dc80-1e14-11eb-8718-c48824e2a040.png)

<img width="557" alt="Screenshot 2020-11-03 at 20 29 50" src="https://user-images.githubusercontent.com/7084049/98032706-b03e8a80-1e14-11eb-82e3-c9259a2c3d62.png">

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension): Chrome extension.

**Comment by timvandermeij** [comment]
Closing since this looks like a duplicate of #12462. Chrome overall just seems severely broken recently, as evidenced by the number of issues linked to that issue and the fact that also our reference tests are showing such problems in Chrome. Hence there is not much we can do here, so please report this to the Chrome team upstream since there appears to have been a serious regression in the browser.

**Comment by clecap** [comment]
@timvandermeij Happy to "report this to the Chrome team upstream", but as I am completely unfamiliar with Chrome development I would appreciated a hint where to find the appropriate place to drop a bit of heat on them ;-)

**Comment by timvandermeij** [comment]
You can report Chromium bugs upstream at https://bugs.chromium.org/p/chromium/issues/list.

**Comment by clecap** [comment]
@timvanermeij Sorry for coming back on this, but I noticed something, which I do not understand and which might be relevant. When I open up a completely fresh, new, extension-empty profile in Chrome and drag-drop my offending PDF file onto it, it renders absolutely correctly.

I am not aware of the internals of Chrome and whether there is another built-in PDF renderer (it looks like it is). **BUT - and that is the crucial point** the problems I reported show up only, when I install an additional extension with pdf.js, such as the PDF viewer extension.

Why would it be a problem with Chrome upstream when I get the error only when a pdf.js extension is plugged in?
